blocks/components/editable/dom-to-react.js

```diff
@@ -0,0 +1,29 @@
+import { createElement } from 'react';
```

Comment:
> That, or we could just commit to individually publishing npm modules for the utilities we create.

If we don't depend on html-to-react anyway, I was thinking of just creating a separate package that has some options to handle nodes (like we need to ignore internal mce stuff).